api-client/src/endpoints/createEventsEndpoint.ts

```diff
@@ -125,7 +132,9 @@ const createRecurringEvents = (event: JsonEventType): JsonEventType[] => {
     })),
     event: {
       ...event.event,
+      start: date.toISOString(),
```

Comment:
Do we really need `start` and `start_date` both?

Reply:
We do not really "need" both, but as this function updates these values for recurring events, i think it makes sense and the code more robust if both values are set correctly.

Reply:
Maybe add a comment explaining that? I definitely see myself at some point attempting to remove one of these 😅

Reply:
Sorry also to ask. I didn't get the purpose yet for it. Where do we want to use `start_date`. Even these attributes are not listed in `JsonEventType[]`

Reply:
Same, we shouldn't need both :see_no_evil: